vendor.conf

```diff
@@ -37,11 +37,18 @@ github.com/opencontainers/runc v1.0.0-rc2
 # OCI runtime spec, latest release as of 12/16/2016
 github.com/opencontainers/runtime-spec v1.0.0-rc3
 # logrus, latest release as of 12/16/2016
+# FIXME: github.com/docker/docker/pkg/archive requires this pkg as github.com/Sirupsen/logrus.
+# So we vendor github.com/Sirupsen as well.
```

Comment:
Looking at https://github.com/docker/containerd/issues/376#issuecomment-267718697 , probably requirement for pkg/archive will be removed (in Phase 2 or 3?). Then this sirupsen-Sirupsen issue will be solved without changes for `docker/docker`.